index.d.ts

```diff
@@ -1,6 +1,7 @@
 /// <reference path="./interfaces.d.ts" />
 
 declare module "cloud-device-emulator" {
-	function getSeverAddress(): Promise<Number>;
+	function getSeverAddress(): Promise<ICloudDeviceServerInfo>;
 	var deviceEmitter: CloudDeviceEmitter;
+	function refresh(deviceIdentifier: string): Promise<string>
```

Comment:
The return type here is marked as `Promise<string>` but the method doesn't return anything